client/components/DrawingGame.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import * as handTrack from 'handtrackjs';
+
+const modelParams = {
+  flipHorizontal: true,
+  // reduce input image size for gains in speed.
+  imageScaleFactor: 0.7,
+  // maximum number of boxes to detect
+  maxNumBoxes: 1,
+  iouThreshold: 0.5,
+  scoreThreshold: 0.75,
+};
+
+let model = null;
+const video = document.getElementById('myVideo');
+const canvas = document.getElementById('canvas');
+const context = canvas.getContext('2d');
+
+export default class DrawingGame extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVideo: false,
+      message: '',
+    };
+    this.startVideo = this.startVideo.bind(this);
+    // this.toggleVideo = this.toggleVideo.bind(this);
+    this.runDetection = this.runDetection.bind(this);
+    this.startGame = this.startGame.bind(this);
+  }
+
+  startVideo() {
+    handTrack.startVideo(video)
+      .then((status) => {
+        console.log('video started', status);
+        if (status) {
+          this.setState({ isVideo: true, message: 'Video started. Now tracking.' });
+          console.log('video started again...');
+          this.runDetection();
+        } else {
+          console.log('please enable video');
+          this.setState({ message: 'Please enable video.' });
+        }
+      });
+  }
+
+  // toggleVideo() {
```

Comment:
do you think we'll need a toggle option in the final product? if someone has to draw, they probably need to have their video turned on the whole time

Reply:
I was playing around with this in order to figure out the best way to have a stop/start drawing function. I don't think I'm going to go this route, but I don't want to remove it until I'm sure.